package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperview",
-  "version": "0.68.0",
+  "version": "0.68.1",
```

Comment:
should we consider a major version bump (1.0.0)? or at least a feature version bump here (0.69.0)?
Also isn't this automatically updated by npm tooling?

Reply:
It will be versioned when released. This change should be undone here.